inc/ThirdParty/Themes/MinimalistBlogger.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace WP_Rocket\ThirdParty\Themes;
+
+class MinimalistBlogger implements \WP_Rocket\Event_Management\Subscriber_Interface {
+
+
+	/**
+	 * Return an array of events that this subscriber wants to listen to.
+	 *
+	 * @return array
+	 */
+	public static function get_subscribed_events() {
+		$events = [];
+		if ( self::is_mb() ) {
+			$events['rocket_delay_js_exclusions'] = 'exclude_jquery_from_delay_js';
+		}
+		return $events;
+	}
+
+	/**
+	 * Excludes some MinimalistBlogger JS from delay JS execution
+	 *
+	 * @since 3.10.2
+	 *
+	 * @param array $exclusions Array of exclusion patterns.
+	 *
+	 * @return array
+	 */
+	public function exclude_jquery_from_delay_js( array $exclusions = [] ) {
+		$exclusions[] = '/jquery-?[0-9.](.*)(.min|.slim|.slim.min)?.js';
+		$exclusions[] = '/jquery-migrate(.min)?.js';
+		return $exclusions;
+	}
+
+	/**
+	 * Checks if the current theme (or parent) is MinimalistBlogger
+	 *
+	 * @since 3.6.3
+	 *
+	 * @param WP_Theme $theme Instance of the theme.
+	 */
+	private static function is_mb( $theme = null ) {
+		$theme = $theme instanceof WP_Theme ? $theme : wp_get_theme();
+
+		return ( str_contains( 'minimalist-blogger', $theme->get_template() ) );
+	}
```

Comment:
I'm thinking for this we should standardize the current theme check across all themes compatibility classes, using an interface and a parent abstract class implementing a method like `is_theme()`, and a class property that would be storing the theme name for each implementing class.